### PR TITLE
DM-15111: NCSA DUO and change to ~/notebooks are browser root

### PR DESCRIPTION
**Pull request by jonathansick**
- Updates the log in docs to take into account NCSA DUO (mostly defers to the NCSA documentation)

- Changes related to `~/notebooks` no longer being privileged as the file browser root.

https://nb.lsst.io/v/DM-15111

### CODE REVIEW

**Review by athornton — CHANGES_REQUESTED** [review]
Turns out that the Jellybean $HOME is /jhome on lsst-dev, accessed via $HOME/jhome .

Other than that, looks good.

File: environment/directories.rst [hunk 1]
```diff
@@ -38,23 +34,20 @@ $HOME
...
 Your home directory is just that.
-It’s backed by essentially the same storage as your home directory on `lsst-dev`_ in the LSST Data Facility.
+
+The :file:`$HOME` directory is also the base directory for the `JupyterLab file browser`_.
+Notebooks and other files need to be inside the :file:`HOME` directory (or a subdirectory) for you to open them from the `file browser`_.
+
+Inside the :file:`$HOME` directory is a :file:`~/notebooks` directory that you can use to organize your personal notebooks.
+This directory also contains the :ref:`~/notebooks/.user_setups <lsst-kernel-user-setups>` file that you can use to configure the environment your notebooks run in.
+
+:file:`$HOME` is backed by essentially the same storage as your home directory on `lsst-dev`_ in the LSST Data Facility.
```
Comment (ktlim) [inline environment/directories.rst]: Note that we will soon (or already do?) have a class of notebook users who are not staff and do not have home directories on `lsst-dev`.

File: getting-started/logging-in.rst [hunk 2]
```diff
@@ -6,33 +6,40 @@ You need NCSA Kerberos credentials to log into the LSST Science Platform Noteboo
...
+Install and configure the NCSA VPN (Cisco AnyConnect and NCSA DUO)
+==================================================================
 
-.. warning::
+To use the notebook aspect from non-approved networks, you need to use the NCSA VPN.
+This requires you to:
 
-   In most browsers, the Java-based VPN installation will fail.
-   Wait for the direct-download fallback to become available and install the AnyConnect client that way.
+- Install the `Cisco AnyConnect client`_ on you computer.
```
Comment (ktlim) [inline getting-started/logging-in.rst]: "you" -> "your"

**Comment by jonathansick** [comment]
screen shot 2018-07-27 at 7 07 06 pm

I think this language in the note should address all the concerns.

**Pull request merged**